app/components/dropdown.hbs

```diff
@@ -0,0 +1,15 @@
+<div data-test-dropdown class="dropdown">
+  <label data-test-label class="label" for={{@name}}>{{@field}}</label>
+  {{#if @required}}
+    <span data-test-required class="required">*</span>
+    <select data-test-dropdown-field class="dropdown__field user-input input-regular" name={{@name}} id={{@name}} required>
+    <option data-test-dropdown-default disabled selected value={{@placeHolder}}>{{@placeHolder}}</option>
+    {{yield}}
+  </select>
+  {{else}}
+  <select data-test-dropdown-field class="dropdown__field user-input input-regular" name={{@name}} id={{@name}}>
+    <option data-test-dropdown-default disabled selected value={{@placeHolder}}>{{@placeHolder}}</option>
+    {{yield}}
+  </select>
+  {{/if}}
```

Comment:
We can optimize here. Check the below comment
https://github.com/Real-Dev-Squad/website-www/pull/380/files#r1067259032

Reply:
At first I had done the same by passing required=true/false but when I was looking into this I got to know whether we are passing required true or false it is behaving same as true(as in image for reference).
Also attaching Stackoverflow reference.

Stack overflow reference - 
https://stackoverflow.com/questions/48651166/validation-required-true-false-html-form

![image](https://user-images.githubusercontent.com/85296770/212158128-ee5401c3-aaa5-4314-8966-273cc553f9ba.png)

Reply:
Yes, I checked this as well but in React it works, not sure about the reason but it's a boolean property and same as disabled and I have worked with disabled property in Ember and it also expects a boolean, so wanted you to give a try for the required property in Ember, if it doesn't work. We will go with your approach.

Reply:
I checked this and yes it's not working but in Ember or in frameworks you can achieve it, and I verified this myself 
In the component :
Name : input.hbs
```
<input type="text" required={{@required}} />
```
Where it is called :
```
<Input @required={{false}} />
OR
<Input @required={{true}} />
```

Reply:
You have to pass the value in curly brackets

Reply:
Yeah got it will do this👍🏻